### DISCUSSION TITLE
Can I help?

### DISCUSSION THREAD

**Discussion by TheCompAce** [TheCompAce/Auto-GPT-Powershell#10] [Q&A]
If you would like to be a developer, please leave a message  here, explaining how you can help, and our team will vote on it. (This being the first post, and only one man the first help I think I need is a Mod, to help keep the development team going, so we have someone to talk to people, because developers are not people persons.)

**Comment by Kallamamran** [comment]
I'd love to help, but I'm afraid I have a life that might interfer to much with frequent moderation?! How much help would you need?

**Comment by TheCompAce** [comment]
I just want to give someone access to delete stuff that is not part of the forum,

**Comment by Kallamamran** [comment]
I'd be happy to help

**Reply by TheCompAce** [reply]
OK, I will learn how to add you to meds. I am mainly doing all this to get use to how GitHub works. So thanks for the challenge.

**Comment by Kallamamran** [comment]
I need to learn more about how GitHub works as well, so thanks for the invite 😊

**Comment by TheCompAce** [comment]
Took me a minute, but welcome aboard.

**Comment by Kallamamran** [comment]
Thanks :)
